### ISSUE TITLE
cannot run with LRA aligner

### ISSUE THREAD

**Issue by xtan1221** [EichlerLab/pav#27]
Hi there, 

Thank you for developing this great tool. I was trying to run the program with LRA as the aligner but could not make it work. Specifically, I am not sure where to put what parameter to change the aligner from minimap2 to LRA. Could you please explain how to do it with example?

Thanks!

**Comment by paudano** [comment]
Sure, add it to `config.json`.

Example `config.json`:
```
{
  "reference": "/path/to/hg38.no_alt.fa.gz",
  "aligner": "lra"
}
```

That's all PAV needs, and it will run with LRA alignments for all samples.

If you are running PAV from Docker or Singularity, LRA is included in the image. If you are running it natively, you'll need LRA installed and findable in your `PATH` environment variable.

**Comment by xtan1221** [comment]
@paudano 
Thank you for the help. I tried to run PAV with the LRA as aligner, but got the following error: 
```
10174732 minimizers with multiplicity smaller than 30
There are 2576677 minimizers left
Waiting at most 40 seconds for missing files.
MissingOutputException in rule data_align_ref_lra_index  in line 73 of /opt/pav/rules/data.snakefile:
Job Missing files after 40 seconds. This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait:
data/ref/ref.fa.gz.mmi completed successfully, but some output files are missing. 16
Removing output files of failed job data_align_ref_lra_index since they might be corrupted:
data/ref/ref.fa.gz.gli
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-01-06T183820.272134.snakemake.log
```

I am not sure if it is due to filesystem latency, so I increased it from 20 to 40 sec and still got the same error.
also I noticed that all the missing files mentioned in the log message actually exist. 

Here is the full log file:
[2023-01-06T183820.272134.snakemake.log](https://github.com/EichlerLab/pav/files/10362353/2023-01-06T183820.272134.snakemake.log)

I wonder if anything I can do to deal with this error? If you need further information, please let me know.

Thanks!

**Comment by paudano** [comment]
I can reproduce this myself, and it doesn't look like it's running out of memory. I think something changed with LRA and it might not be writing the ".gli" index file. If so, I'll change PAV so it doesn't expect the ".gli" index. I have messaged Mark Chaisson (LRA PI and developer) for clarification.

Filesystem latency is usually a problem with distributed jobs and slow shared storage (written by a compute host on shared storage, then the Snakemake process on another host has to wait for it to appear). Thanks for troubleshooting!

**Comment by paudano** [comment]
The fix has been pushed in version 2.2.4 (up on Singularity and Docker). I tested, PAV ran to completion with LRA alignments. I am closing this, but please re-open if you have any further issues. Thank you for reporting this!
